### PR TITLE
Update docs links and CNCF Slack links

### PR DESCRIPTION
**Pull request by kai-tillman**
## Description

Slack is moving to the CNCF Slack.  Updating links

### CODE REVIEW

**Review by tenshinhigashi — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ You can also use Helm to install Emissary. For more information, see the instruc
 
 Check out full the [Emissary
 documentation](https://www.getambassador.io/docs/emissary/) at
-www.getambassador.io.
+www.getambassador.io/docs/open-source.
```
Comment (dotslashderek) [inline README.md:68]: I know it's not your change but I think "the" and "full" are inverted here 😄

**Review by dotslashderek — COMMENTED** [review]
Hey your changes look fine @kai-tillman I found an unrelated potential doc grammar type issue, didn't know if you wanted to fix it here.

File: README.md (hunk 1 above)
Comment (kai-tillman) [reply README.md:68]: Yeah, I'll fix that now

**Review by dotslashderek — APPROVED** [review]
Looks good 🚀

**Pull request merged**